mmv1/products/databasemigrationservice/connectionprofile.yaml

```
@@ -0,0 +1,576 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ConnectionProfile'
+self_link: 'projects/{{project}}/locations/{{location}}/connectionProfiles/{{connection_profile_id}}'
+import_format: ["projects/{{project}}/locations/{{location}}/connectionProfiles/{{connection_profile_id}}"]
+base_url: 'projects/{{project}}/locations/{{location}}/connectionProfiles'
+create_url: 'projects/{{project}}/locations/{{location}}/connectionProfiles?connectionProfileId={{connection_profile_id}}'
+update_verb: :PATCH 
+update_mask: true
+description: 'A connection profile definition.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Database Migration': 'https://cloud.google.com/database-migration/docs/'
+  api: 'https://cloud.google.com/database-migration/docs/reference/rest/v1/projects.locations.connectionProfiles/create'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 60
+      update_minutes: 60
+      delete_minutes: 60
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+autogen_async: true
+examples:
+    - !ruby/object:Provider::Terraform::Examples
+      name: "database_migration_service_connection_profile_cloudsql"
+      primary_resource_id: "cloudsqlprofile"
+      ignore_read_extra:
+        - "mysql.0.password"
+        - "mysql.0.ssl.0.ca_certificate"
+        - "mysql.0.ssl.0.client_certificate"
+        - "mysql.0.ssl.0.client_key"
+      vars:
+        sqldb: "my-database"
+        sqldb_cert: "my-cert"
+        sqldb_user: "my-username"
+        sqldb_pass: "my-password"
+        from_profile: "my-fromprofileid"
+        to_profile: "my-toprofileid"
+    - !ruby/object:Provider::Terraform::Examples
+      name: "database_migration_service_connection_profile_postgre"
+      primary_resource_id: "postgreprofile"
```

```suggestion
      name: "database_migration_service_connection_profile_postgres"
      primary_resource_id: "postgresprofile"
```
minor detail, but I think it's referred to either postgreSQL or postgres